api/allIncidents/allincidentsRouter.js

```diff
@@ -0,0 +1,200 @@
+const express = require('express');
+const router = express.Router();
+const Incidents = require('./incidentsModel');
+const {
+  checkIncidentExists,
+  validateAndSanitizeIncidentQueries,
+} = require('../middleware');
+
+// TODO document shape of objects coming and going
+
+// ''' ---------> Incidents Routes <--------- '''
+// ### GET /showallincidents ###
+// - returns all incidents in the BE database
+// ⬇️ swagger docs code generation ⬇️
+/**
+ * @swagger
+ * /allincidents:
+ *  get:
+ *    summary: path returning all incidents in database in reverse chronological order
+ *    tags:
+ *      - allincidents
+ *    produces:
+ *      - application/json
+ *    responses:
+ *      200:
+ *        description: success ... returns an array of incident objects
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              required:
+ *                - api
+ *              properties:
+ *                data:
+ *                  type: array
+ *                  example: [
+  {
+    incident_id: 1,
+    incident_date: '2021-03-01T00:00:00.000Z',
+    tweet_id: '1366291653267513344',
+    user_name: 'shafiur',
+    desc:
```

Comment:
desc -> description

Reply:
fixed and corrected, weird because I copied it directly from seed file

Reply:
aaaaanddd fixed in seeds as well